install.sh

```diff
@@ -4,18 +4,24 @@
 set -e
 
 #chromebrew directories
-OWNER="${OWNER:-skycocker}"
-REPO="${REPO:-chromebrew}"
-BRANCH="${BRANCH:-master}"
-URL="https://raw.githubusercontent.com/${OWNER}/${REPO}/${BRANCH}"
-CREW_PREFIX="${CREW_PREFIX:-/usr/local}"
+
+: "${OWNER:=skycocker}"
+: "${REPO:=chromebrew}"
+: "${BRANCH:=master}"
+: "${CURL:=curl}"
+: "${CREW_PREFIX:=/usr/local}"
+: "${CREW_CACHE_DIR:=$CREW_PREFIX/tmp/packages}"
+
+URL="https://github.com/${OWNER}/${REPO}"
 CREW_LIB_PATH="${CREW_PREFIX}/lib/crew"
 CREW_CONFIG_PATH="${CREW_PREFIX}/etc/crew"
 CREW_BREW_DIR="${CREW_PREFIX}/tmp/crew"
 CREW_DEST_DIR="${CREW_BREW_DIR}/dest"
 CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
-CURL="${CURL:-curl}"
-CREW_CACHE_DIR="${CREW_CACHE_DIR:-$CREW_PREFIX/tmp/packages}"
+
+# Set sparse-checkout folders and include install.sh for use in reinstalls
+# or to fix problems.
+sparse_checkout=(bin/ lib/ packages/ install.sh)
```

Comment:
Does this work? (So we get the same stuff checked out as before?)
`sparse_checkout=(bin/ lib/ packages/ tools/ crew install.sh)`

Reply:
Does we need to checkout `crew`? (it is a symlink)